x-pack/plugins/apm/public/components/app/correlations/correlations_log.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiAccordion, EuiCode, EuiPanel } from '@elastic/eui';
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+
+interface Props {
+  log: string[];
+  displayLog?: boolean;
+}
+export function CorrelationsLog({ log }: Props) {
+  return (
+    <EuiAccordion
+      id="apmCorrelationsLogAccordion"
+      buttonContent={i18n.translate('xpack.apm.correlations.logButtonContent', {
+        defaultMessage: 'Log',
+      })}
+    >
+      <EuiPanel color="subdued">
+        {log.map((d, i) => {
+          const splitItem = d.split(': ');
+          return (
+            <p key={i}>
+              <small>
+                <EuiCode>{splitItem[0]}</EuiCode> {splitItem[1]}
```

Comment:
```suggestion
                <EuiCode>{asAbsoluteDateTime(logMessage.timestamp)}</EuiCode> {logMessage.message}
```

Reply:
Btw I just noticed that we need a few tests for `asAbsoluteDateTime`. Can you added the following to `x-pack/plugins/apm/common/utils/formatters/datetime.test.ts`

```tsx
    it('milliseconds', () => {
      moment.tz.setDefault('Europe/Copenhagen');
      expect(asAbsoluteDateTime(1559390400000, 'milliseconds')).toBe(
        'Jun 1, 2019, 14:00:00.000 (UTC+2)'
      );
    });

    it('seconds', () => {
      moment.tz.setDefault('Europe/Copenhagen');
      expect(asAbsoluteDateTime(1559390400000, 'seconds')).toBe(
        'Jun 1, 2019, 14:00:00 (UTC+2)'
      );
    });

    it('minutes', () => {
      moment.tz.setDefault('Europe/Copenhagen');
      expect(asAbsoluteDateTime(1559390400000, 'minutes')).toBe(
        'Jun 1, 2019, 14:00 (UTC+2)'
      );
    });

    it('hours', () => {
      moment.tz.setDefault('Europe/Copenhagen');
      expect(asAbsoluteDateTime(1559390400000, 'hours')).toBe(
        'Jun 1, 2019, 14 (UTC+2)'
      );
    });
```

Thanks!

Reply:
Updated here https://github.com/elastic/kibana/pull/109187/commits/82d5457f4696584969059f9ad78a897722507d2c